### ISSUE TITLE
Feature : fix use class

### ISSUE THREAD

**Issue by mehdi-fathi** [imanghafoori1/laravel-microscope#24]
I check out this package.it's great. I think feature auto-fix use class in use class is great. if you think like me. I like to add this feature.

**Comment by imanghafoori1** [comment]
processing grouped imports is complicated, but yeah it is nice to have it at some certain levels.
and only after bug fixes for current features.

**Comment by imanghafoori1** [comment]
This feature is implemented in v1.0.93
